AndroidCISetupGuide.md

```diff
@@ -1,87 +1,215 @@
 
 ## Android CI Setup Guide
 
-### Overview
-
-There are two Github action workflows that are integrated to our projects. One of them provides a build check after push. It's called as **Push Build Test**. The other one is responsible for deploying app to Google Play Internal App Distribution Channel. This workflow is called as **Deploy Release to Google Play** and works after pushing tags to any branch. This can be restricted to master branch later on with two line change on the workflow file.
-
-## Branch Build Test
-This workflow will be triggered after every push unless it's tag branch. The workflow can be integrated in any development phase of the project but needs to be customized according to **productFlavors** on the app.
-
-```yaml
-name: Branch Build Test
-
-on:
-  push:
-    branches-ignore:
-      - 'refs/tags/**'
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@master
-      - name: set up JDK 1.8
-        uses: actions/setup-java@master
-        with:
-          java-version: 1.8
-      - name: Check If App Builds At All Flavors
-        run: sudo ./gradlew :app:assembleProdDebug :app:assembleProdRelease :app:assembleStagingDebug :app:assembleStagingRelease
-      - name: Find APKs
-        run: sudo find . -iname '*.apk'
-```
+### :eye: Overview
+
+There are three Github action workflows that are integrated to our projects. One of them provides a build check after push. It's called as **Commit Check**. Second one is responsible for deploying app to Google Play Internal App Distribution Channel. This workflow is called as **Deploy To Firebase** and works after pushing tags with (-) in it (For example 1.0.1-3) to any branch. Third one is responsible for generating App Bundle for deploying to Google Play Store. This workflow is called as **Deploy Store Release** and works after pushing tags without dash (-) in it (For example 1.0.1) to any branch.
```

Comment:
You may want to change Github action workflows here.

Reply:
And also, deploy store release is not responsible of deploying to Google Play Store.